### PR TITLE
Bash script

### PR DESCRIPTION
**Pull request by solker**
#34

The file structure on the custom repository after run the script in first time. 
selection_083

### CODE REVIEW

File: scripts/init_shoov.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+git clone https://github.com/amitaibu/gizra-behat .shoov-temp
+cd .shoov-temp
+cp .shoov.yml ../
+
+
+tar zxf behat/behat-vendor.tar.gz
+cp -R behat ../
+cp ../behat/behat.local.yml.example ../behat/behat.local.yml
```
Comment (amitaibu) [inline scripts/init_shoov.sh:15]: `echo you can now run the behat test using cd behat && ./bin/behat`

**Comment by solker** [comment]
It's script: [gist](https://gist.githubusercontent.com/solker/3999cd675b16c47329a5/raw/6c0364fd1a0cc39f296a921ad3b41a2dd6bb644c/init_ci_build)

``` bash
#!/usr/bin/env bash

# Test that git repository exist.
if [ ! -d .git ]; then
  echo 'Script should be running in exist git repository.'
  exit 1
fi

# Name of temporary directory.
TEMP_DIR='.shoov-temp'

# Clone another repository with example shoov.yml and behat tests.
git clone https://github.com/amitaibu/gizra-behat $TEMP_DIR

# Move .shoov.yml to root of repository.
cp $TEMP_DIR/.shoov.yml ./

# Unpack behat enviroment.
tar zxf $TEMP_DIR/behat/behat-vendor.tar.gz -C $TEMP_DIR/behat

# Move behat enviroment with tests to root directory.
mv $TEMP_DIR/behat ./

# Remove temporary folder.
rm -rf $TEMP_DIR

# Create a behat local config.
cp ./behat/behat.local.yml.example ./behat/behat.local.yml

# Show git status?
# Make new commit?
# Push to server?

echo 'Shoov is configured. Now you can activate your repository on shoov.io.';
echo 'Also you can run tests by next command: cd behat && bin/behat'

exit 0
```

You can test it by execute next command in your repository:

`curl -sL http://bit.ly/1KPt5UE | sh`

Full URL is `https://gist.githubusercontent.com/solker/3999cd675b16c47329a5/raw/6c0364fd1a0cc39f296a921ad3b41a2dd6bb644c/init_ci_build`

**Comment by amitaibu** [comment]
> Make new commit?

Yes, lets commit (but not push)

**Comment by amitaibu** [comment]
I'm getting an error:

``` bash
...
Checking connectivity... done
mv: rename .shoov-temp/behat to ./behat: Directory not empty
Shoov is configured. Now you can activate your repository on shoov.io.
```

**Comment by solker** [comment]
Closed in favor of https://github.com/shoov/shoov.io/pull/3

**Pull request closed without merging**